Classes/ViewHelpers/Show/ValueViewHelper.php

```diff
@@ -36,12 +36,15 @@ public function render() {
 		$fieldName = $this->templateVariableContainer->get('fieldName');
 
 		if ($value instanceof ObjectStorage) {
-			$object = $value->current();
-
 			// special case for file reference which is ok to be hardcoded
-			if ($object instanceof FileReference) {
-				$value = $object->getOriginalResource()->getName();
+			$objectsArray = $value->toArray();
+			$value = array();
```

Comment:
Please a different name as $value here since we already use it above: possibly $names or $fileNames.